src/ansys/mapdl/core/launcher.py

```diff
@@ -308,8 +308,10 @@ def launch_grpc(
         Number of processors.  Defaults to 2.
 
     ram : float, optional
-        Fixed amount of memory to request for MAPDL.  If ``None``,
-        then MAPDL will use as much as available on the host machine.
+        Specifies the total size of the Workspace (memory) in
+        megabytes used for the initial allocation. Specify a negative number to
+        force a fixed size throughout the run. If ``None``, then the default is
+        2 GB (2048 MB).
```

Comment:
```suggestion
        Total size in megabytes of the workspace (memory) used for the initial allocation.
        The default is ``None``, in which case 2 GB (2048 MB) is used. To force a fixed size
        throughout the run, specify a negative number.
```
In the brief description, omit the leading article (the, an, a). This should be done throughout someday. All sentences following should be complete. Place the default first, followed by adding information. Someday, I'll try to get to editing existing MAPDL doc to meet our doc standards.